### PR TITLE
Archive logs onto artifacts server

### PR DESCRIPTION
**Pull request by p3ck**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -264,6 +264,12 @@ podTemplate(name: podName,
                         pipelineUtils.convertProps(ostree_props, ostree_props_groovy)
                         load(ostree_props_groovy)
 
+                        // Rsync push logs
+                        env.rsync_paths = "logs"
+                        env.rsync_from = "${env.WORKSPACE}/ci-pipeline" + currentStage
```
Comment (scoheb) [inline Jenkinsfile]: missing a '/' ?

File: Jenkinsfile [hunk 2]
```diff
@@ -330,6 +336,11 @@ podTemplate(name: podName,
                         } else {
                             echo "Not Pushing a New Image"
                         }
+                        // Rsync push logs
+                        env.rsync_paths = "logs"
+                        env.rsync_from = "${env.WORKSPACE}/" + currentStage
```
Comment (scoheb) [inline Jenkinsfile]: same as above?

Comment (johnbieren) [reply Jenkinsfile]: image-compose isn't in dir(ci-pipeline)

**Comment by johnbieren** [comment]
[test]

**Comment by p3ck** [comment]
[test] f27 fixed?

**Comment by johnbieren** [comment]
[test]

**Comment by p3ck** [comment]
[test] jenkins issues

**Comment by p3ck** [comment]
[test] try again...

**Comment by p3ck** [comment]
[merge]

**Pull request merged**